minifi-toolkit/minifi-toolkit-configuration/src/test/java/org/apache/nifi/minifi/toolkit/configuration/dto/ProcessorSchemaTest.java

```diff
@@ -45,6 +45,35 @@
     private final String testKey = "testKey";
     private final String testValue = "testValue";
     private final String testPenalizationPeriod = "55 s";
+    private final String testAnnotationData = ">&lt;criteria&gt;\n" +
```

Comment:
Very minor, but Is the beginning '>' a typo? I realize it's just test data, but seems like it would be invalid.